front/lib/api/assistant/actions/tables_query.ts

```diff
@@ -145,13 +156,13 @@ export class TablesQueryAction extends BaseAction {
     }
 
     if (hasResultsFile) {
-      const attachment = getTablesQueryResultsFileAttachment({
+      const attachments = getTablesQueryResultsFileAttachments({
```

Comment:
This is purely aesthetic rigth?

Reply:
This is indeed because now we can have 2 attachments!